lib/components-schema-v1_3_x.ts

```diff
@@ -0,0 +1,581 @@
+                                            type: 'string',
+                                            description: 'Metadata property name. The field is case sensitive. Slashes should be used to separate levels',
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+                required: ['name', 'label', 'icon'],
+                additionalProperties: false,
+            },
+        },
+
+        componentProperties: {
+            type: 'array',
+            description: 'List of available component properties',
+            items: {
+                type: 'object',
+                required: ['name', 'label', 'control', 'dataType'],
+                additionalProperties: false,
+                properties: componentPropertyDefinition,
+            },
+        },
+
+        groups: componentGroupDefinition,
+
+        conversionRules: {
+            type: 'object',
+            description: 'Conversion rules for transforming one component into another component',
+            additionalProperties: {
+                type: 'object',
+                additionalProperties: {
+                    anyOf: [
+                        {
+                            enum: ['auto'],
+                            description: 'Map one component into another component by auto matching fields',
+                        },
+                        {
+                            type: 'object',
+                            description: 'Map one component into another component by field mapping',
+                            properties: {
+                                type: {enum: ['simple']},
+                                map: {
+                                    type: 'object',
+                                    additionalProperties: {type: 'string'},
+                                },
+                            },
+                            required: ['type', 'map'],
+                        },
+                        {
+                            type: 'object',
+                            properties: {
+                                type: {enum: ['from-container']},
+                                container: {type: 'string'},
+                            },
+                            required: ['type', 'container'],
+                        },
+                        {
+                            type: 'object',
+                            properties: {
+                                type: {enum: ['to-container']},
+                                container: {type: 'string'},
+                            },
+                            required: ['type', 'container'],
+                        },
+                    ],
+                },
+            },
+        },
+
+        conversionShortcutComponents: {
```

Comment:
We could also do something like:

{code}
shortcuts: {
    conversionComponents: { .... }
}
{code}

But that only makes sense if we expect more configuration for shortcuts in the future and want to keep it grouped under one object.